### PR TITLE
PR to merge feature branch into master (newsweek.com)

### PR DESCRIPTION
**Pull request by UmairAhmad125**
This PR for newsweek.com (Using Bootstrap) solo HTML/CSS project.

This project includes a responsive clone of the newsweek.com website.

To achieve responsiveness bootstrap and media queries are used. The use of custom CSS and media queries is kept minimal.
This Project was manually tested.
More details and live link to the project is included in the readme file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1598 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```
Comment (HADDADSOHAIB) [inline index.html]: The style.css should be the last imported file

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @UmairAhmad125     :handshake: 

### Good job so far :clap:  your project looks very good and similarto the original :medal_sports: , you did great job on the design.
### There is some issues that need to be fixed before I could approve the project:
- I see that you have stickler files but I don't see stickler checks on PR, please head to stickler-ci.com and enable stickler.
Screenshot from 2020-06-11 01-17-51
No stickler checks.
- Please fix validators errors and warnings:
Screenshot from 2020-06-11 01-18-45
- Your project is not good at small width:
Your project:
Screenshot from 2020-06-11 01-22-01
Screenshot from 2020-06-11 01-22-31
Screenshot from 2020-06-11 01-23-13
Screenshot from 2020-06-11 01-23-26
Screenshot from 2020-06-11 01-23-41
As you can see from 575px, your page is broken, please fix and remember that your project should be responsive at all size.
### Once you are sone with the changes, ask for another code review.

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required! :recycle: 
Hi @UmairAhmad125, great job using a lot of bootstrap classes since that is the goal of this project. :+1: However, there are some changes you need to consider in order to finish this project. These include:
- [ ] __[OPTIONAL]__ There is unnecessary white space above the navbar. Removing it would make your clone look more like the original.
- [x] On large screens, your `OPINIONS` section has too much white space between the images and corresponding text.
- [x] On large screens, your `In the magazine`, `Editor's picks`, and `US, WORLD, TECH & SCIENCE` sections appear more centered than the rest of the page. This isn't how the original site looks like.
- [x] The [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) should be used for the __dominant__ content of the `<body>` of a document.  In your HTML, you have used it to do exactly what the `<body>` tag is already doing. It is more semantic to put your `<nav>` and `<footer>` outside your `<main>` tag.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi @UmairAhmad125 ! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Pull request merged**